guide/how-it-works/custom-spending-conditions.md

```diff
@@ -0,0 +1,225 @@
+
+### Backups
+
+In terms of securing the wallet, backing up the individual private keys is not enough. The wallet configuration, also known as the wallet descriptor, needs to be backed up as well. Without this “map”, you will not be able to restore any multi-key wallet.
+
+## Pros & cons
+
+#### Pros
+
+- Can increase fault tolerance, thereby reducing the risk of lost funds.
+- Can reduce the risk of theft.
+- Enables non-custodial social recovery and inheritance workflows.
+- Can improve security and flexibility for shared wallet usage in organizational settings.
+
+#### Cons
+
+- Miniscript is not yet widely supported by software and hardware wallets.
+- More complex setup and backup processes.
+- It can be cumbersome to manage timelocks, if the wallet contains many UTXOs.
+
+## Best practice
+
+#### When to use
+
+- When storing large amounts.
+- When funds need to be accessed by several people or an organization.
+- When the target audience is likely to own hardware wallets.
+- When users are likely to be very knowledgeable or be guided through setup and use
+
+#### When not to use
+
+- For small amounts.
+- When users are likely to be new to bitcoin.
+
+#### Do
+
+- **Provide real-world examples or templates** for common setups to help users choose and implement the right configuration for their needs.
+- **Provide informational content to help users guide through the setup process** and educate them about how to properly use a time-based wallet. They can be powerful tools to achieve very robust setups. However, with an increasing number of conditions, setups become complex very quickly, which might actually decrease their security.
+- **Educate users that composing condition layers is only one aspect of their setup.** Make them aware that they should think through the adjacent topics before implementing a specific scheme. These topics include aspects like the trust relationships with other users, key backups and wallet configuration backups. The more layers people use, the more complicated the entire setup gets.
+- **If possible, provide a way to save unfinished wallet configurations during the setup process.** As the creation process may involve many steps and/or users might not have all the necessary signing keys available, they would need to start from scratch at a different point in time, which could cause frustration.
+- **Use Taproot in your product**, as it adds many privacy benefits to the users, especially in a multi-sig context. First, using multi-sig on its own reduces the anonymity set. Taproot helps mitigate this aspect, because only the executed part is revealed on-chain. Second, spending conditions that are not used in a Taproot transaction are not revealed on-chain. This means that inactive keys or conditions cannot be detected through chain-surveillance tools. Third, Taproot transactions have a smaller on-chain footprint. Therefore, they are more economical.
```

Comment:
```suggestion
- **Use Taproot in your product**, as it adds many privacy benefits to the users, especially in a multi-key context. First, using multi-key on its own reduces the anonymity set. Taproot helps mitigate this aspect, because only the executed part is revealed on-chain. Second, spending conditions that are not used in a Taproot transaction are not revealed on-chain. This means that inactive keys or conditions cannot be detected through chain-surveillance tools. Third, Taproot transactions have a smaller on-chain footprint. Therefore, they are more economical.
```
The rest of the page uses "multi-key".

Can also link Taproot to the [address page in the glossary](https://bitcoin.design/guide/glossary/address/#taproot-address---p2tr).